alacritty.yml

```diff
@@ -531,7 +532,7 @@
 # be mapped to the `ReceiveChar` action. Alternatively, you can use `None` for
 # a no-op if you do not wish to receive input characters for that binding.
 #key_bindings:
-  # (Windows/Linux only)
+  # (Windows and Linux only)
```

Comment:
I feel like it'll be a good idea to separate windows only bindings and provide a common section for Linux and Windows bindings.